extensions/amp-viqeo-player/0.1/amp-viqeo-player.js

```diff
@@ -188,6 +191,27 @@ class AmpViqeoPlayer extends AMP.BaseElement {
     return true; // Call layoutCallback again.
   }
 
+  /** @override */
+  createPlaceholderCallback() {
+    const placeholder = this.element.ownerDocument.createElement('amp-img');
+    this.propagateAttributes(['aria-label'], placeholder);
+    if (placeholder.hasAttribute('aria-label')) {
+      placeholder.setAttribute('alt',
+          'Loading video - ' + placeholder.getAttribute('aria-label')
+      );
+    } else {
+      placeholder.setAttribute('alt', 'Loading video');
+    }
+    placeholder.setAttribute('src',
+        `http://cdn.viqeo.tv/preview/${encodeURIComponent(this.videoId_)}.jpg`);
+    placeholder.setAttribute('layout', 'fill');
+    placeholder.setAttribute('placeholder', '');
+    placeholder.setAttribute('referrerpolicy', 'origin');
+    placeholder.setAttribute('alt', 'Loading video');
```

Comment:
this is overriding alt again, I will fix it and merge, no action needed from you.

Reply:
thanks